src/components/ReportActionItem/ReportPreview.js

```diff
@@ -113,6 +117,7 @@ function ReportPreview(props) {
     const iouCanceled = ReportUtils.isArchivedRoom(props.chatReport);
     const numberOfRequests = ReportActionUtils.getNumberOfMoneyRequests(props.action);
     const moneyRequestComment = lodashGet(props.action, 'childLastMoneyRequestComment', '');
+    const isReportDraft = ReportUtils.isReportDraft(props.iouReport);
```

Comment:
I would probably add this logic here `ReportUtils.isPolicyExpenseChat(props.chatReport)` and then update the usages in the file to make sure we dont call this for IOU reports no reason

Then we would have:
```
const isReportDraft = isPolicyExpenseChat && ReportUtils.isReportDraft(props.iouReport)
```